src/mock-dnas/mockData.js

```diff
@@ -27,13 +27,14 @@ const noteEntries = {
 const data = {
   notes: {
     notes: {
-      create_note: ({ note_spec }) => {
-        const id = 1
-        const address = 'QmRccTDUM1UcJWuxW3aMjfYkSBFmhBnGtgB7lAddress00'
+      create_note: ({ note_input: noteInput }) => {
+        const id = Object.keys(noteEntries).length + 1
+        const address = 'QmRccTDUM1UcJWuxW3aMjfYkSBFmhBnGtgB7lAddress0' + id
+        noteEntries[address] = { id, ...noteInput }
```

Comment:
@robbiecarlton - Can you remind me of the issues were?  I remember there being a decision between three different approaches, but there were many changes since then...

Reply:
never mind